---
title: "The Missing Lncs"
Shorttitle: An exploratory study of lncRNAs part in muscle adaptation
output:
  word_document:
    toc: yes
    reference_docx: templates/wordtemplate.docx
  html_document:
    toc: yes
    df_print: paged
  pdf_document:
    toc: yes
bibliography: references_lncs.bib
editor_options:
  chunk_output_type: console
csl: templates/apa.csl
---

\pagebreak

# Acknowledgement


\pagebreak

# Abstract

#### Title

#### Intention
#### Research question 

#### Method
#### Results 
#### Conclusion 
#### Keywords 

\pagebreak

# Abbreviations
Differentially expressed (DE)

Eukaryotic translation initiation factor 4E-binding protein 1 (4E-BP1)

Focal adhesion kinase (FAK)

Fold change (FC)

Growth arrest specific 5 (GAS5)

Long non coding RNA (lncRNA)

Mammalian target of rapamycin (mTOR)   

Mammalian target of rapamycin complex 1 (mTORC1)  

Mammalian target of rapamycin complex 2 (mTORC2)

Mitogen-activated protein kinase (MAPK)

Mitogen-activated protein kinase kinase (MAP2K)

Mitogen-activated protein kinase kinase kinase (MAP3K)

Muscle protein synthesis (MPS)

Phosphatidic acid (PA)

Protein Kinase B (Akt)

Quantitative /real time polymerase chain reaction (qPCR)

Ribosomal protein S6K (p70S6K)

RNA component of mitochondrial RNA processing endoribonuclease (RMRP)

40S ribosomal protein S6 kinase (RSK)

\pagebreak

# I. Theory
Humans are made for movement, and prolonged exercise training can enhance human exercise performance and help to maintain good health. Proper function of skeletal muscles are important for every aspect in life. Athletes need strong and enduring muscles to perform their best in competitions. Elderly people need strong muscles in the extremities to be able to walk across the road on green light, and to ascend stairs. Strong leg muscles are also important to prevent loss of balance and injury. When the muscles are exposed to mechanical stimuli, they adapt and becomes stronger or more persevering. Usually one tries to induce mechanical stimuli to the muscles with strength training. There are a vast number of different strength training programs and methods, and one can vary the training volume with sets, repetitions or load [@Egan2013]. The training volume is closely connected to increased muscle strength, and usually one finds large increase in muscle strength after high intensity training studies. How many sets and repetitions to apply, are a never ending discussion. The most used volume variables when performing strength training are probably 3 sets and 8-12 repetitions.
Much is known about the different strength training methods and how to maximize strength and muscle gain. Tailormade strength training bouts are big business, and are used for athletes, rehabilitation patients and exercisers alike. But little is known about what really happens when the muscle adapts to training. What are the rationales behind the small bashful changes in RNA expression, protein translation and other similar processes. The main signalling patways, like Akt and MAPK, are known. But the knowledge of crosstalk and internal signaling are inadequate. One exemple are the lncRNAs and their function, another are miRNAs.




Long non coding RNAs (lncRNAs) are a bit of a mystery. For a long time they were solely regarded as transcriptional noise, but recently they were found interesting. [@Kung2013]. LncRNAs and their implication on muscle adaptation to strength training may be vast.**Kort benevnemulige funksjoner først** Some is known, and cancer research advocates more research on lncRNAs.**Vær mer generell – trenger du skrive cancer research her?** Especially the ones that are connected to tumor growth [@Hu2018]. Little is kown about the lncRNAs role in skeletal muscle adaptation to strength training. In studies on mice, the lncRNA H19 induces muscle differentiation. The expression of lncRNAs in human skeletal muscles are mostly unknown. The knowledge of their function, and how they react to strength training are scarce if any [@Hughes2018].



## I.I Training volume and muscle growth
In the initial phase of **?** have low hypertrophy effect from strength training, and the strength gain mostly comes from neural adaptations to muscle activation [@Folland2007]. Adaptation to strength training is an individual **variation** response and **?** are not identified or understood yet. What is known, is that one can vary load, sets and repetitions to target either CSA or neuromuscular drive [@Folland2007]. One usually vary the load between 1RM and 10RM, the repetitions between 4-12 and the sets between 1-6 [@Fry2004] Muscle growth is highly correlated with strength training volume  For a long time this was debated, but recent meta-studies have shown that the former is true. **elaborate** [@Ralston2017].

Strength training induces mechanical stimuli to skeletal muscles and the muscle adapts, among others, by adding sarcomeres in parallel in muscle fibres[@Folland2007]. Strength training results in neural adaptations, improved strength, alter muscle phenotype. and increased cross sectional area (CSA) of the muscle fiber . Increased RFD (rate of increase in force at the contraction onset)[@Maffiuletti2016]. Other adaptations to strength training is increased CSA**har skrevet før, rydd opp**, specially IIa fibres, increase in noncontractile tissue, e.g collagen, and change in muscle fibers angle of pennation. Adaptations to strength training shows after 8-12 weeks of repeated training[@Folland2007]. Over time, the strength gain will be more due to muscle growth than neural adaptations. There will be an increase in muscle net protein synthesis (MPS) . MPS is increased due to lower protein degradation and higher, maybe more efficient, protein synthesis [@Damas2016]. The central neural component is important for muscle adaptations due to strength training, especially with unilateral training. With the latter the CSA in the untrained leg does not change, but one can observe an increase in strength [@Munn2004].

Systems biology is collaboration across different research fields and sciences[@Hester2011]. In need of multilevel studies to make a "Human Model". Trying to collect data from e.g. biology, physiology and **data?** to predict how the human body responds and adapts to various stimuli. This aproach to understand adaptation to strength training advocates that there are a strong correlation between training volume and muscle growth [@Hester2011].  **elaborate**


## I.II Cellular pathways
Different signalling pathways that leads to muscle adaptation after strength training have been identified. **Ha gjerne en bredere inngang til dette kapittelet – mange pathways, etc** One of the most important to strength training adaptations is mammalian target of rapamycin (mTOR) [@Hoppeler2016]. MTOR is a part of the phosphatidylinositol 3-kinase-related kinase family and has two distinct multi-protein complexes mtor c1 and c2. The former is recognized as the one most important to muscle adaptation. We don't know much about the function of mtorc2, but it looks**På generellt grunnlag: du har en del «muntlige» vendinger underveis i teksten – slik som denne – prøv å bli kvitt disse** like it may be associated with the ribosome and regulating ribosomal activity, and cell survival [Chaillou2014]. mTORC1 is targeting different signalling pathways and proteins, and is probably very important for muscle protein synthesis (MPS) [@Mirzoev2018]. mTORc1 is activated by focal adhesion kinase (FAK) - Akt - mTOR- torc1- p70S6K. mTORC2 is downstream from FAK, Akt and upstream from eukaryotic translation initiation factor 4E-binding protein 1 (4E-BP1). Initiation of protein translation is activated by phosphatidic acid (PA), the latter activates mTORC1 and thereafter ribosomal protein S6K (p70S6K). Mechanical stress activates mitogen-activated protein kinase (MAPK), which can activate myc and phosphorylate C-myc.**Myc = familie av gener (c-myc, l-Myc og n-Myc)**
**C-myc = medlem av Myc-familien**
 Regulates transcription factors and therby may regulate transcription of proteins [@Hoppeler2016]. Little is known of proteins that regulates satellite cells. @McCroskery2003 advocates that satellite cells are regulated by myostatin. Upregulated myostatin levels increases p21, a cyclin-dependent kinase inhibitor, and inhibits differentiation. Satellite cells, or muscle stem cells, are small cells that can evolve to sketelal muscle cells. When activated they can proliferate and transform to myoblasts.  The latter can induce muscle fibre hypertrophy or make new muscle cells [@Morgan2003]. They are situated between the sarcolemma and membrane of the muscle fibre. Activated upon mechanical strain. Excercise triggers a cascade of different signaling molecules, e.g. growth factors and cytokines. HGF activates satellite cells, fibroblast growth factor (FGF) and insulin-like growth factor-I (IGF-1) increases proliferation.

adaptive hypertrophy mainly driven by MPS (muscle protein synthesis) after activation of mTOR, ribosomal protein S6K (p70) and downstream variables [@Bodine2001].

Translation of ribosome and the making of proteins is the key to muscle growth. Translation depends on two variables, translation capacity and translation efficiency. Translation capacity is all about **muntlig språk**how many ribosome are available, tRNA and translation factors . And translation efficiency is how efficient the ribosome does the protein synthesis [Chaillou2014]. Increase in the latter is likely one of the main variables  behind elevated MPS as a response to strength training [@ONeil2009].



mTorC1 activity is regulated by several upstream regulators. One of them is phosphoinositide 3-kinase (PI3K), that again regulates protein kinase B (Akt) [@Bodine2001]. Mechanical stimuli leads to that Insulin-like growth factor 1 (IGF-1) binds to receptors in the cell membrane, and this initiate stimulation of PI3K- and Akt-activity [@McCarthy2010]. AKT phosporylates downstream effectors and that activates G-protein Rheb (Ras-homolog enriched in brain) and that again activates mTORC1.

Ras is activated when extracellular mitogen binds to the membrane receptor. Followed by activation of Mitogen-activated protein kinase kinase kinase(MAP3K), Mitogen-activated protein kinase kinase (MAP2K) and Mitogen-activated protein kinase (MAP). The latter can then activate Myc, or other transcription factors [@McCarthy2010]. Induce cell growth and proliferation.

Mdm2 -p53 stress response pathway regulates cellular homesostasis. If activated it results in apoptosis, cell cycle arrest, DNA repair or replicative senescence. It is important to regulate cell growth [@Bartlett2014].**Elaborate**

## I.I Long non-coding RNA
### I.III.I Overwiev 
  
The lncRNAs, as the name implies, are very long.**Nope – 200 baser er kort – om enn mye lenger enn microRNA (som er referansepunkt for definisjonen)** They are made up of over 200 nucleotides [@Ponting2009].  Little is known of their function and the evidence for lncRNAs function is scarce. But due to more advanced research methods, and higher interest the recent years, many lncRNAs has been identified and annotated. 

**Starte setningen ved å presisere “first**»In the early 1990s, @Brannan1990 discovered that the lncRNA H19 was involved in epigentic regulation. Later, many more lncRNAs has been identified and described. Their function in epigenetic regulation are still eluded, but some of them are well known. Some of the lncRNAs can alter the coding gene by pairing with mRNA [@Wang2016], and other can interact with microRNAs and  make them miss their target mRNA [@Cesana2011]. Yet another group of lncRNAs can encode micropetides that are shorter than 100 amino acids and by doing that alter the micropeptide induced functions [@Douglas2015]. Lncrnas have a hefty toolbox to use, and most of the lncs directly linked to myogenesis acts as transcriptional or epigenetic regulators [@Li2018].

LncRNAs can be allocated to five categories based on where they are situated in the genome:
1) sense, 2) antisense, 3)bidirectional, 4)intronic and 5)intergenic [@Ponting2009].
In the first category they overlap one or more exons of another exon on the same strand. The second is the same as the first, except the lncRNA is on the opposite strand. In the third category, the lncRNA is in close genomic proximity to a coding transcript on the opposite strand. The fourth is when it is derived from an intron on a second transcript, and the fifth is when it is in the genomic interval between two different genes. Some of the lncRNAs functions can be seen in figure 1.
  
  **Figure from [@Kung2013] shows some of the lncRNAs functions**

  
### I.III.II Cell growth and proliferation

Some of the most known lncRNAs induces cell growth and proliferation, and are important in cancer research. Many knockout gene studies have been conducted to explore lncRNAs role in tumor growth. 
In animal studies, one has discovered that lncRNAs may regulate satellite cell biology [@Li2018]. Satellite cells are important for muscle regeneration.  If an injury occurs, the satellite cells will be activated and become myoblasts. Pax7 is downregulated and myogenic regulatory factors (MRFs) are activated to start cell differentiation, thereby making new muscle fibers and replenish the damaged muscle cells [@Kuang2007]. Another important feature for the lncRNAs is that they play a role in epigenetic and transcriptional regulation of chromatins. The Lncs interacts with chromatins, and may inhibit other transcriptional regulators activities [@Han2014].

Some of the lncRNAs can alter the coding gene by pairing with mRNA [@Wang2016], and other can interact with microRNAs and  make them miss their target mRNA [@Cesana2011]. Yet another group of lncRNAs can encode micropetides that are shorter than 100 amino acids and by doing that alter the micropeptide induced functions [@Douglas2015]. Lncrnas have a hefty toolbox to use, and most of the lncs directly linked to myogenesis acts as transcriptional or epigenetic regulators [@Li2018].

 In mice, lncRNA SYISL interacts with polycomb repressive complex 2 and regulates myogenesis [@Jin2018] It's human homolog is lncRNA AK021986, but no ensemble ID has been allocated and the latter is therefore removed from the qPCR analysis. H19 lnc in mice muscles [@Kallen2013].



Some of the known lncRNAs and their function
The lncRNA Growth arrest specific 5 (GAS5) supresses MYC translation [@Pickard2013]. (Parrot/LINP1) is a positive regulator of c-Myc and ribosomal biogenesis [@Zhang2016]. (PVT1) is activated in the early phase of muscle atrophy. PVT1 alter mitochondrial respiration, myofiber size, apoptosis and mito/autophagy [@Tseng2014]. RNA component of mitochondrial RNA processing endoribonuclease (RMRP) alters the transport trough the mitochondrial membrane [@Wang2018]. (Linc-MD1) has many functions related to muscle adaptations. It regulates myogenic differentiation, myogenesis and hypertrophy. Linc-MD1 downregulates myogenic markers when depleted. It is reported to influence the mRNA levels of miRNA-targeted muscle differentiation genes, by and blocking the target mRNA for miR-133 and miR-135 [@Cesana2011].
  
  

  
### I.III.III Methods used to study lncRNAs in human muscle cells
As mentioned, little is known of lncRNAs function and expression in human biology. And especially in muscle biology. How their expression changes dependent on training stimuli, are largely unknown. The lncRNAs may alter signaling patways and cellular responses. By altering signaling patways, they may alter muscle growt and differentiation.


Discovery and characterization of lncRNAs has sped up due to the recent years leap in high throughput gene sequencing technology[@Reuter2015]. 

Use qPCR or RNA seq. Which one is best. RNA seq yields massive data. 

Not all lncRNA has poly A tail. This is essential to be able to sequence.**Gjelder polyA-primed seq** @Sun2016 defined 7692 lncRNAs in bovine skeletal muscle using Ribo_Zero RNA-seq. This sequencing technology can identify both poly A+ and poly A- transcripts. A poly A tail consists of many adenosine monophosphates and helps preventing degradation of mRNA. LncRNAs have usually low expression and are highly tissue specific. They usually have a poly A+ or poly A- tail at thee 3' end of the transcript. [@Cabili2011]

Only six of the nine lncRNAs that underwent qPCR analysis was recognized in the sequence data.

sequencing depth


The lack of knowledge on how lncRNAs adapts to mechanical stimuli, strength training, in human muscle, advocates further research on the topic. The goal of this study was to explore and identify lncRNAs in human muscle cells.**Her er du konkret – da ville jeg henvist til muscle biopsies--Og å undersøke deres responser på akutt og langtids styrketrening** The research questions are as follows:
1) How manye lncRNAs can be idetified from the rna sequence data **Vær konkret – hvor mange lncRNAs can be identified in RNA-se data from m.vastus lateralis.**




2) How many differentially expressed (DE) lncRNAs can be identified from gene sequencing data pool after a strength training bout.**Gjør dette om til ett research question (evt to – ett for akutte effekter og ett for langtidseffekter) – vær konkret -i.e. bruk effekter av to og tolv uker heavy-load strength training på uttrykket av LncRNA**
3) How many DE lncRNAS increases? **Få inn research question på volumperspektivet**
4) How many DE lncRNAS decreases

5) Is there a correlation between qPCR data and seq. data.



\pagebreak

# 1. Introduction

**flere kilder**

Humans are made for movement, and prolonged exercise training can enhance human exercise performance and help to maintain good health. Proper function of skeletal muscles are important for every aspect in life. Athletes need strong and enduring muscles to perform their best in competitions. Elderly people need strong muscles in the extremities to be able to walk across the road on green light, and to ascend stairs. Strong leg muscles are also important to prevent loss of balance and injury. When the muscles are exposed to mechanical stimuli, they adapt and becomes stronger or more persevering. **Poenget her bør snarere være at muskelfunksjoner kan bedres gjennom styrketrening **Usually one tries to induce mechanical stimuli to the muscles with strength training. There are a vast number of different strength training programs and methods, and one can vary the training volume with sets, repetitions or load [@Egan2013]. The training volume is closely connected to increased muscle strength, and usually one finds large increase in muscle strength after high intensity training studies. How many sets and repetitions to apply, **are a never ending discussion-skriv om**. **poenget bør snarere være at vi vet overraskende lite om dette perspektivet (dvs vi vet overraskende lite om hvilke treningsprogammer som bidrar til optimale treningsresponser**

The most used volume variables when performing strength training are probably 3 sets and 8-12 repetitions.
Much is known about the different strength training methods and how to maximize strength and muscle gain. Tailormade strength training bouts are big business, and are used for athletes, rehabilitation patients and exercisers alike. BUt little is known about what really happens when the muscle adapts to training. What are the rationales behind the small bashful changes in RNA expression, protein translation and other similar processes. THe main signalling patways, like Akt and MAPK, are rather well characterized. But the knowledge of crosstalk and internal signaling are inadequate.**altså – vi vet lite om hvordan aktiviteten i disse pathways er regulated – avhenger av integrated regulation of a range of different classes of organic molecules, incl proteins, RNA, DNA etc. – kom deretter inn på lncRNA som eksempel – a large group of RNAs som har varierte funksjoner i celler – vi vet lite om disse, men de ser ut til å spille viktige roller i mange cellulære funksjoner, varierende fra cell proliferatin and differentiation to cellular housekeeping funksjoner som for eksempel metabolism, etc** One exemple are the lncRNAs and their function, another are miRNAs.




 Long non coding RNAs (lncRNAs) are a bit of a mystery. For a long time they were solely regarded as transcriptional noise, but recently they were found interesting. [@Kung2013]. LncRNAs and their implication on muscle adaptation to strength training may be vast.**Kort benevnemulige funksjoner først** Some is known, and cancer research advocates more research on lncRNAs.**Vær mer generell – trenger du skrive cancer research her?**  Some is known, and cancer research advocates more research on lncRNAs. Especially the ones that are connected to tumor growth [@Hu2018]. Little is kown about the lncRNAs role in skeletal muscle adaptation to strength training. In studies on mice, the lncRNA H19 induces muscle differentiation. The expression of lncRNAs in human skeletal muscles are mostly unknown. The knowledge of their function, and how they react to strength training are scarce if any [@Hughes2018].


**lncRNAs expression**
**human and animals studies**
**cellular growth**





In this study 9 lncRNAs were analysed with qPCR. Gene sequencing data was also obtained and analysed together with qPCR data. Gene sequencing data is an vast pool and this is an exploratory study.  

The lack**se kommentarer I hovedintro – du er primært ute etter tre ting: i) kartlegge LncRNA-uttrykk i skjelettmuskel hos mennesker, ii) kartlegge generelle LncRNA-responser på styrketrening,og iii) sammenligne effekter av styrketrening med ulikt volum på LncRNA-uttrykk** of knowledge on how lncRNAs adapts to mechanical stimuli, strength training, in human muscle, advocates further research on the topic. The goal of this study was to explore and identify lncRNAs in human muscle cells. The research questions are as follows:
1) How manye lncRNAs can be idetified from the rna sequence data.
2) How many differentially expressed (DE) lncRNAs can be identified from gene sequencing data pool after a strength training bout.
3) How many DE lncRNAS increases?
4) How many DE lncRNAS decreases

5) Is there a correlation between qPCR data and seq. data.






# 2. Methods
This study is based on the 1/3 set study completed by @Hammarström2020.


## 2.1. Ethical approval
Information about potential discomforts and risks associated with the study were given to all the participants and they gave their informed consent before study enrolment. All procedures were performed in accordance to the Declaration of Helsinki. The study design was pre-registered (ClinicalTrials.gov Identifier: NCT02179307) and approved by the local ethics committee at Lillehammer University College, Department of Sport Science (no. 2013-11-22:2).


## 2.2. Intervention and participants overview 
Forty one female and male participants were recruited to the study. **25 personer  hadde komplette biopsisett – disse ble sekvensert**The eligibility criteria were age between 18 and 40 years and non-smoking. The exclusion criteria were impaired muscle strength due to ongoing or previous injury, prescribed medicine that could alter exercise adaptations, more than one strength exercise bout weekly during the last 12 months or local anaesthetic intolerance. 7 participants were excluded during data analysis due to different reasons. Details can be found in @Hammarström2020. 

The intervention was conducted as 12 weeks of full-body strength training.
Within-participants differentiation of training volume was achieved by doing unilaterally leg exercises. The strength-training consisted of one set and three set exercises, and the exercise was randomly allocated to the participants two legs. Assessment of muscle strength was performed at baseline and week 3, 5, 9 and after the intervention. Body composition was assessed before and after the intervention and dietary data was reported 4-5 executive days after week 6. M.vastus lateralis biopsies were taken at the following 4 different timepoints bilaterally: week 0 (rested state), week 2 (pre-exercise), week 2 (post-exercise, acute-phase) and week 12 (rested state)(figure2). The detailed study protocol can be found in @Hammarström2020


**figure 2 training intervention**



 
 

## 2.3. Training protocol
The training bouts always started with 5 min of ergometer cycling with Borgs RPE 12-14 as a warm up.Thereafter 4 bodyweight exercises (sit-ups, push-ups, back-extensions and squats) with 10 repetitions each. Followed by 10 reps at 50% of 1 repetitions maximum (1RM) for each strength exercise. Thereafter unilateral leg press, leg curl and knee extension either as one set or three sets for the latter. After the lower leg exercises they performed two sets of pull-down, seated rowing or shoulder-press and bench press. The intensity was progressed from 10RM (2 weeks), 8RM (3 weeks) to 7RM (7weeks). The rest period between the latter sets was 90-180 seconds. For more details see @Hammarström2020


## 2.4. Muscle biopsies
The muscle biopsies were taken, within 10 minutes, bilaterally from m. vastus lateralis using a spring-loaded biopsy instrument (Bard Magnum, Bard, Rud, Norway) with a 12-gauge needle (Universal-plus, Medax, San Possidonio, Italy). Local anaesthetics (Xylocaine, 10 mg ml−1 with adrenaline 5µgml−1, AstraZeneca AS,Oslo,Norway) was used during the latter protocol. The resting samples were taken after a standardised meal, at the same timepoint in the morning. Biopsies pre exercise (5 session) was taken 2 days after the 4th session. Post exercise biopsies were taken 3-6 days after intervention ended. Patella and spina iliaca anterior superior (SIAS) was used as landmarks, and the first biopsy was taken from 1/3 of the latter distance. Consecutive biopsies were taken 2 cm proximal to the latter sample. Ice cold saline solution (0,9%) was used when dissecting the muscle samples free from connective tissue and blood. Thereafter 15mg muscle tissue were moved to 4% formalin solution for later immunohistochemistry testing. The muscle tissue (60 mg) that were to be used in RNA- and protein-analysis were quickly frozen in isopentane and stored at -80 degrees Celsius   



## 2.5. Total RNA extraction
RNA extraction was done in accordance with the protocol found in @Hammarström2020 article. 1 ml of TRIzol reagent (Invitrogen, Life technologies AS, Oslo, Norway) was used to homogenise about 25 mg of wet muscle tissue. O.5 mm RNase-free zirconium oxide beads was added togehter with an exogenous RNA Control (λ
polyA External Standard Kit, Takara Bio Inc., Shiga,
Japan) to the solution and run in a Bullet blender (Bullet Blender,NextAdvanced,Averill Park,NY, USA). 400 μl of the phase was allocated and isopropanol was used to precipitate a RNA pellet. Before the RNA pellet was eluted in TE buffer, 70% EtOH was used in three washing steps. A spectrophotometer determined quality and amount of RNA.


## 2.6. RNA sequencing data

RNA_sequencing was performed in accordance to the unpublished paper @Khan2020.

LncRNAs were identified with BiomaRT R package and Ensemble ID. With this method is it not possible to find lncRNAs without Ensemble id. The latter applied to AKO21986, one of the lncRNAs analyzed with qPCR, and it was removed from further analysis.


Parrot/LINP1 annotated with LINP1

AKO21986 was sorted out because does not have ensemble number.



Only 16 of the lncRNAs identified in the gene sequence data have entrezgene gene id number. Entrezgene ID is needed to perform a gene ontology analysis. National Center for Biotechnology Information (NCBI) hosts the gene specific databbse Entrez Gene. The database generates unique and stable gene identifier integers [@Maglott2011]. This gene ID are then used to integrate different information about the specific gene, such as nomenclature, sequence, pathways and protein interaction [@Maglott2011]. The information in the database is based on results from NCBI's other databases. Research on lncRNAs and their functions are therefore important and needed to evolve the database and increase the number of anotated lncRNAs.



## 2.7. Quantitative real-time reverse transcription polymerase chain reaction (qPCR) 

In this study, the qPCR analysis was performed to validate RNA-seq data and find lncRNAs that corresponded in both anaalysis modalities.

The qPCR was done with a qPCR machine (Applied Biosystems 7500 fast Real-Time PCR Systems, Life Technologies AS). Used 384 well plates filled with total 10 µl solution. The latter consisting of 2 µl cDNA, specific primers (Forward and Reverse, total 1µl), H20 and a prepared master mix (2X SYBR Select Master Mix, Applied Biosystems, Life Technologies AS). The qPCR protocol was 40 cycles (3 s 95°C denaturing and 30 s 60°C annealing).
Primertest were performed and the primers with the best melt-curves was selected (single product amplification)

### 2.7.1. Overview of primers.
Primers were designed for all selected long non coding RNAs (LINCs) with Primer3Plus [@Untergasser2012] and ordered from Thermo Scientific.

A primertest was conducted to validate which primers to use in the qPCR setup. The primerpair with the best meltcurve, with no biproduct or primerdimers, were choosen.

**table primers choosen**












## 2.8. Statistics
All data-analysis and wrangling was done in R (rcore2016).
LncRNAs were identified using biomaRT package in R. Mixed-effects negative binomial count models were fitted and saved in 
./R/dge_list_models.R. Results saved in RDS files for easy loading. The fixed effects are reduced
to only contain gene-specific time + time:sets.
The RNAseq data was normalized to tissue weight, according to [@Khan2020].

Statistical significance was set to α = 0.01 and significant FC were below -0.5 and above 0.5.
Fold change analysis was done at all timepoints and between low and moderate training volume.

Raw data was exported from the qPCR machine and uploaded to RStudio and analysed with the qpcR-package (Ritz & Spiess, 2008) written for R (R Core Team, 2018). Treshold cycles (Ct) were estimated within the latter. Gene expression data were log-transformed prior to statistical analysis.

A Correlation between qpcr data and RNAseq was performed using Pearson test. 

All files and code can be found at github.com "Innlevering_masteroppgave"



# 3. Results
This study is based on the muscle samples from @Hammarström2020. They reported**ikke reported – 12 uker med styrketrening førte til… ** a strenght gain of ~25% against ~19% after twelve weeks of strength training from moderate and low volume training respectively. Muscle hypertrophy**bruk lean-body mass** correspond to the latter with an increase of ~3.5% in the moderate volume group and ~2.0% in the low volume [@Hammarström2020]. **fremvises disse data I figure/tabell? Bruke tabell**
 

## 3.1. LncRNAs identified 
In total 15025 genes were identified in the RNA sequencing data from m. vastus lateralis, and 1390 of them were identified as lncRNAs (Apendix 1). With the α set to 0.01 and the fold change significance level set to < -0.5 : >0.5, no significant**statistikken skal ikke gjentas her – beskrives i metode.Start med å presentere data for research question 1, etterfulgt av 2 og 3 – dvs volume analyses tas til slutt**
difference was found between low and moderate volume. The same applied to the data when adjusting α level to 0.05 and 0.1. 


Of the 1390 lncRNAs, 169 were differentially expressed (DE) at timepoint w2pre. 164 lncRNAs were upregulated and 5 were downregulated (figur 3A). At timepoint w2post 102 lncRNAs were DE, and 40 were upregulated and 62 downregulated (figur 3C). After the training period, at timepoint w12, 64 lncRNAs were upregulated (figure 3E). The lncRNAs were arranged according to adjusted p-value and FC. The top ten from their respective timepoint were choosen and explored further (figur 3B, 3D and 3F respectively). 17 of the DE lncRNAs were DE across all timepoints (figur 3G). No significant difference were found between low and moderate volume at any timepoint (figur4). Of the top ten lncRNAs, only 3 are downregulated (figure 3C).**Du må presentere eksempler på gener – alt som skal brukes i diskusjonen, må også nevnes i results – henvis først til cellulære funksjoner – e.g. LncRNA involved in regulatin of cellular proliferation… gener som tidligere har blitt funnet å inneha spesielle funksjoner i skjelettmuskel fremheves spesielt (gjelder seff H19, men også LincMD1, og evt andre).** 

**Du bør også nevne at genene som ble regulert akutt stort sett er ikke-karakteriserte (kan i diskusjonen brukes til å understreke hvor lite vi vet i dag)**



topten w2pre
Zhou et al. (2007) [@Zhou2007]
Qian et al. (2018)[@Qian2018]
Wu et al. (2018)[@Wu2018]
Watanabe et al. (2008)[@Watanabe2008]
Kallen et al. (2013)[@Kallen2013]
Zheng et al. (2018)[@Zheng2018]
Wencheng et al. (2018)[@Wencheng2018]



topten w2post
Hu et al. (2016)[@Hu2016]


topten w12
Yu et al. (2015)[@Yu2015]
Sirey et al. (2019)[@Sirey2019]
Lessard et al. (2015)[@Lessard2015]
Yang et al. (2016)[@Yang2016]
Lingadahalli et al. (2018)[@Lingadahalli2018]
Herter et al. (2019)[@Herter2019]

**used normalizing factor of 2.874087. Found it by summarise mean tissue weight, by calculating RNA.pr.mg from conc*elution.volume) / prot.mrna1**


**The p-value of the test is 0.03229, which is less than the significance level alpha = 0.05. We can conclude that** 
**seq and qpcr are significantly correlated with a correlation coefficient of -0.85 and p-value of 0.03229**


# 4. Discussion
No significant**Presenter hovedfunnene dine I same rekkefølge som tidligere – først generelt LncRNA-uttrykk, deretter endringer i uttrykk som respons på styrketrening, tilslutt volumperspektivet. I dette første avsnittet presenteres kun hovedlinjer – diskusjon rundt funksjonene til for eksempel H19 tas seinere i diskusjonen (i.e. hva kan økt H19-uttrykk tenkes å føre til?)** difference between low and moderate volume training was detected in the data analysis. Most of the DE lncRNAs detected were upregulated. The total number of De lncRNAs found are lower than others have found [@Sun2016]. Little is known of lncRNAs function in muscle tissue, and few articles have studied DE across different timepoints after an training intervention period. LincMD1 may be important for muscle biology and adaptation to strength training. It was regognized in this study as a DE lncRNA, but was not among the top ten genes at any timepoint. H19 is another interesting lncRNA, and the latter is part of the top ten genes. H19 is known to accelerate muscle differentiation in mice [@Kallen2013].

**Elaborate on top ten genes**

 
1390 lncRNAs were identified with RNAseq in muscle tissue and somewhere between 10 and 15 percent were differentially expressed. The above may implie that lncRNAs alter muscle adaptation and affects different signalling pathways.




lncRNAs did not differ in response to different volume modalities. No difference were found between low and moderate training volume. This is in accordance with the findings of @Khan2020. They describes very similar responses to low and moderate volume. In the latter study, the samples were taken from the contralateral leg. The intrabiological model advocates high correlations between repeated samples. 



*3)	Tids-effekter LNC-responser. Mange endret seg (utgjør xx % av alle gener som endret uttrykk). Diskutere spesielt interessante gener i detalj (som fremhevet i diskusjonens første avsnitt) – kan vi si noe om biologiske effekter? Bruk gjerne litteratur på andre vevsmodeller enn muskel*

      @Yang2016 discovered a higher number of lncRNAs                                 both up and downregulated


*4)	Metodiske utfordringer og fremtidige perspektiver*
Few lncRNAs with entrezgene ID made it difficult to conduct a proper gene ontology analysis. Most of the lncRNAs have hgnc numbers, but little is known of their function and the studies are scarce at best.

To rule out coding RNA, the ensemble database was used. A search was made, and only genes anotated as lncRNAs were choosen. This method may be biased and important lncRNAs that are not anotated, may be wrongfully sorted out.

Sequencing depth determines how many genes that are regognized. One may stipulate that more lncRNAs could be discovered if the sequencing deptt was adjusted. The RNA sequencing method used only recognizes polya lncRNAs **elaborate**



@Khan2020 compared different RNA seg data normalizing aproaches. They advocates tissue weight normalizing, and this study utilizes that aproach. Except from when normalizing acute data. Minimal muscle growth are expected after only one training bout, and lib size normalizing of the data are therefore sufficient. 
3

LncRNAs are gaining status, and more research are conducted on their different functions. In cancer research, knockout studies of different lncRNAs shows promising treatment paths. 

The expression of different lncRNAs may also be used for predictive purposes. It may be possible to predict the adaptation to specific strength trainning based on expression of lncRNAs that induces muscle growth or muscle differentiation. Knockout studies are difficult to conduct on humans, and therefore many studies mice.  




# 5. Conclusion

         
                          
                        

**LINC01405 LINP1  LINCMD1  GAS5 RMRP**

\pagebreak


# References








